src/Devices/TemperatureControl.h

```diff
@@ -0,0 +1,44 @@
+#include <Arduino.h>
+
+// class temp control -temp setpoint +get/set temp target +virtual updateCtrl(),  subclass heater (respective code) and
```

Comment:
Please make comment a little more readable and put it in this format:
```
/**
 * My comment line 1
 * line 2
 */
```

Reply:
This does look like notes that may have been helpful during development but should be removed or expanded.